backend/services/esweService.ts

```diff
@@ -105,72 +105,102 @@ export default class EsweService {
+      const mathsGrade = getLatestGrade(learnerLatestAssessments, AssessmentQualificationType.Maths)
+      const digitalLiteracyGrade = getLatestGrade(learnerLatestAssessments, AssessmentQualificationType.DigitalLiteracy)
 
-    const englishGrade = getLatestGrade(learnerLatestAssessments, AssessmentQualificationType.English)
-    const mathsGrade = getLatestGrade(learnerLatestAssessments, AssessmentQualificationType.Maths)
-    const digitalLiteracyGrade = getLatestGrade(learnerLatestAssessments, AssessmentQualificationType.DigitalLiteracy)
+      const createSkillAssessmentSummary = (learnerAssessment: curious.LearnerAssessment) => {
+        const { qualification, establishmentName } = learnerAssessment || {}
+        const { qualificationType, qualificationGrade, assessmentDate } = qualification || {}
 
-    const createSkillAssessmentSummary = (learnerAssessment: curious.LearnerAssessment) => {
-      const { qualification, establishmentName } = learnerAssessment || {}
-      const { qualificationType, qualificationGrade, assessmentDate } = qualification || {}
+        if (!assessmentDate) {
+          return [
+            {
+              label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
+              value: 'Awaiting assessment',
+            },
+          ]
+        }
 
-      if (!assessmentDate) {
         return [
           {
-            label: qualificationType,
-            value: 'Awaiting assessment',
+            label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
+            value: qualificationGrade,
+          },
+          {
+            label: 'Assessment date',
+            value: readableDateFormat(assessmentDate, CURIOUS_DATE_FORMAT),
+          },
+          {
+            label: 'Assessment location',
+            value: establishmentName,
           },
         ]
       }
 
-      return [
-        {
-          label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
-          value: qualificationGrade,
-        },
-        {
-          label: 'Assessment date',
-          value: readableDateFormat(assessmentDate, CURIOUS_DATE_FORMAT),
-        },
-        {
-          label: 'Assessment location',
-          value: establishmentName,
-        },
-      ]
-    }
-
-    const functionalSkillLevels = {
-      english: createSkillAssessmentSummary(englishGrade),
-      maths: createSkillAssessmentSummary(mathsGrade),
-      digiLit: createSkillAssessmentSummary(digitalLiteracyGrade),
+      skillLevels = {
+        english: createSkillAssessmentSummary(englishGrade),
+        maths: createSkillAssessmentSummary(mathsGrade),
+        digiLit: createSkillAssessmentSummary(digitalLiteracyGrade),
+      }
+    } catch (e) {
+      if (e.response?.body?.errorCode === 'VC500') {
+        log.info(`Offender record not found in Curious.`)
+        skillLevels = {
```

Comment:
Not that important, but it might be worth moving this to a const at the top of this file with a name like `DEFAULT_SKILL_LEVELS`?